extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -94,13 +94,17 @@ export abstract class AzureAuth implements vscode.Disposable {
 		} catch (ex) {
 			if (ex instanceof AzureAuthError) {
 				if (loginComplete) {
-					loginComplete.reject(ex.getPrintableString());
+					loginComplete.reject(ex);
 				} else {
 					vscode.window.showErrorMessage(ex.getPrintableString());
 				}
 			}
 			Logger.error(ex);
```

Comment:
So why wouldn't you reject with the exception here?

Reply:
I don't really know what other errors can show up, so I don't want them to be sent to browser which is outside this apps control. Could potentially contain PII.